docs/sources/enterprise/reporting.md

```diff
@@ -106,48 +106,26 @@ When the CSV file is generated, it is temporarily written to the `csv` folder in
 A background job runs every 10 minutes and removes temporary CSV files. You can configure how long a CSV file should be stored before being removed by configuring the [temp-data-lifetime]({{< relref "../administration/configuration/#temp-data-lifetime" >}}) setting. This setting also affects how long a renderer PNG file should be stored.
 
 ### Scheduling
+> Note: Scheduler has been significantly changed in Grafana Enterprise v8.1.
 
-Scheduled reports can be sent on a monthly, weekly, daily, or hourly basis. You may also disable scheduling for when you either want to send it via the API.
+Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals. You can also disable scheduling by selecting **Never**: for example, if you want to send the report via the API.
 
-All scheduling indicates when the reporting service will start rendering the dashboard. It can take a few minutes to render a dashboard with a lot of panels.
+{{< figure src="/static/img/docs/enterprise/reports_scheduler_8.1.png" max-width="500px" class="docs-image--no-shadow" >}}
 
-#### Hourly
 
-Hourly reports are generated once per hour. All fields are required.
+**Send now or schedule for later**
 
-- **At minute -** The number of minutes after full hour when the report should be generated.
-- **Time zone -** Time zone to determine the offset of the full hour. Does not currently change the time in the rendered report.
+- **Send now** sends the report immediately after you save it. To stop sending the report at some point in the future, add an end date. If you leave the end date empty, the report is sent out indefinitely.
 
-#### Daily
+- **Send later** schedules a report for a later date, so you must set a start date and time. If you leave the end date empty, the report is sent out indefinitely.
```

Comment:
```suggestion
- **Send later** schedules a report for a later date, if you set a start date and time. If you leave the end date empty, the report is sent out indefinitely.
```
The sentences cause an error in logic.

Reply:
I meant to say that because this options sends report later on a date, the date and time are required.